AmazonChimeSDKDemo/AmazonChimeSDKDemo/CaptionsModel.swift

```diff
@@ -48,18 +49,11 @@ class CaptionsModel: NSObject {
                     })
                 }
                 
-                var entities = [String]()
-                
-                alternative.entities?.forEach({ entity in
-                    entity.content.split(separator: " ").forEach { word in
-                        entities.append(String(word))
-                    }
-                })
-                
+                let entities = alternative.entities?.flatMap { $0.content.components(separatedBy: " ")}
```

Comment:
Change to this, so you don't need force unwrap at line 56

`let entities = alternative.entities?.flatMap { $0.content.components(separatedBy: " ")} ?? []`

Reply:
will update.